server/resp/src/test/java/org/infinispan/server/resp/RespSingleNodeTest.java

```diff
@@ -534,4 +534,23 @@ public void testGetdelNotPresent() {
       String key = "getdel-notpresent";
       assertThat(redis.getdel(key)).isNull();
    }
+
+   @Test
+   public void testStrlen() {
+      RedisCommands<String, String> redis = redisConnection.sync();
+      String key = "strlen";
+      String val = "Hello";
+      String app = " World";
```

Comment:
This test needs to test non ASCII characters as well, and verify the behavior with REDIS server. Unfortunately, https://redis.io/commands/strlen/ doesn't state what it does for non UTF-8 characters if it counts the bytes or the characters. (I have a feeling it may be the latter).

Reply:
Added test. Redis strings are sequence of bytes, so it counts bytes:
```
127.0.0.1:6379> set euro "€"
OK
127.0.0.1:6379> strlen euro
(integer) 3
```